CONTRIBUTING.md

```diff
@@ -0,0 +1,284 @@
+* **Test what you need to**: Write useful tests. Don't write tests that aren't useful. We _generally_ aren't strict about unit testing every line of code. We don't want you to waste your time. But at the same time:
+   * Most new features should have at least one minimal [example](https://github.com/bevyengine/bevy/tree/main/examples). These also serve as simple integration tests, as they are run as part of our CI process.
+   * The more complex or "core" a feature is, the more strict we are about unit tests. Use your best judgement here. We will let you know if your pull request needs more tests. We use [Rust's built in testing framework](https://doc.rust-lang.org/book/ch11-01-writing-tests.html).
+
+## How we're organized
+
+@cart is, for now, our singular Benevolent Dictator and project lead.
+He makes the final decision on both design and code changes within Bevy in order to ensure a coherent vision and consistent quality of code.
+
+In practice, @cart serves as a shockingly accountable dictator: open to new ideas and to changing his mind in the face of compelling arguments or community consensus.
+Check out the next section for details on how this plays out.
+
+[Bevy Org members](https://github.com/orgs/bevyengine/people) are contributors who have:
+
+1. Have actively engaged with Bevy development
+2. Have demonstrated themselves to be polite and welcoming representatives of the project with an understanding of our goals and direction.
+3. Have asked to join the Bevy Org. Reach out to @cart on Discord or [email us](bevyengine@gmail.com) if you are interested. Everyone is welcome to do this. We generally accept membership requests, so don't hesitate if you are interested!
+
+Some Bevy Org members are also [Triage Team](https://github.com/orgs/bevyengine/teams/triage-team) members. These people can label and close issues and PRs but do not have merge rights or any special authority within the community. Existing Bevy Engine Org members can [automatically request membership](https://github.com/orgs/bevyengine/teams/triage-team/members). Once again, if you are interested don't hesitate to apply. We generally accept membership requests.
+
+We heavily limit who has merge rights within the org because this requires a large amount of trust when it comes to ethics, technical ability, and ability to enforce consistent project direction. Currently, only @cart can merge every class of change. @mockersf is allowed to merge small "uncontroversial" changes, provided they have two approvals.
+
+## How we work together
+
+Making a game engine is a huge project and facilitating collaboration is a lot of work. At the moment @cart is our only paid contributor, so [go sponsor him!](https://github.com/sponsors/cart).
+
+While we have *many* active contributors, herding all of these ~~cats~~ birds in a predictable way is challenging.
+
+Bevy releases are intended to be spaced 6-8 weeks apart and tend to target one or two major features, led by @cart.
```

Comment:
This seems untrue in practice, is it an actual goal?

Reply:
Yeah this got less true over time. I think we should just cut it.